gulpfile.js

```diff
@@ -46,7 +47,11 @@ gulp.task('lint', function() {
 });
 
 pipes.buildAngular = function() {
-  return gulp.src('public/angular/js/app.js')
+  var b = browserify({
+    entries: 'public/angular/js/app.js'
+  }).bundle();
+  return b
+    .pipe(source('public/angular/js/app.js'))
     .pipe(plumber())
     .pipe(sourcemaps.init())
     .pipe(browserify())
```

Comment:
I went down the whole `gulp` + `browserify` rabbit hole that you did, and saw the `vinyl-source-stream` [example code](https://github.com/hughsk/vinyl-source-stream) for linking browserify with gulp which matches most of what you have here. Awesome!

One comment: Do we need to call `browserify` at the end of this function chain anymore? I'm not even sure we could, since you replaced the `gulp-browserify` plugin with the actual `browserify` package.

I am probably missing something. Let me know your thoughts!